app/controllers/admin/users_controller.rb

```diff
@@ -1,2 +1,29 @@
 class Admin::UsersController < Admin::BaseController
+  expose :user
+  expose :interests, from: :user
+  expose_decorated(:users, -> { User.all })
+
+  def create
+    user.password = 'secret'
```

Comment:
Why the password is always `secret`? You can use a random stirng generator or just allow user to specific one